aquarius/events/processors.py

```diff
@@ -61,19 +61,16 @@ def __init__(
         self.purgatory = purgatory
         self._chain_id = chain_id
 
-    def check_permission(self):
-        if not os.getenv("RBAC_SERVER_URL"):
+    def check_permission(self, publisher):
```

Comment:
can you please name this publisher_address for more clarity?

otherwise LGTM :)